code/game/objects/items/mail.dm

```diff
@@ -0,0 +1,205 @@
+
+/// Mail is tamper-evident and unresealable, postmarked by CentCom for an individual recepient.
+/obj/item/mail
+	name = "mail"
+	gender = NEUTER
+	desc = "An officially postmarked, tamper-evident parcel regulated by CentCom and made of high-quality materials."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail_small"
+	inhand_icon_state = "paper"
+	worn_icon_state = "paper"
+	item_flags = NOBLUDGEON
+	w_class = WEIGHT_CLASS_SMALL
+	drop_sound = 'sound/items/handling/paper_drop.ogg'
+	pickup_sound =  'sound/items/handling/paper_pickup.ogg'
+	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
+	/// destination tagging for the mail sorter
+	var/sortTag = 0
+	/// who this mail is for and who can open it
+	var/mob/recipient
+	/// how many goodies this mail contains
+	var/goodie_count = 1
+	/// goodies which can be given to anyone.
+	/// the base weight for cash is 56. for there to be a 50/50 chance of getting a department item, they need 56 weight as well.
+	var/list/generic_goodies = list(
+		/obj/item/stack/spacecash/c100 = 30,
+		/obj/item/stack/spacecash/c200 = 20,
+		/obj/item/stack/spacecash/c500 = 10,
+		/obj/item/stack/spacecash/c1000 = 5,
+		/obj/item/stack/spacecash/c10000 = 1
+	)
```

Comment:
So, you'll probably want to tilt this far downwards for the moment, since 100 c is about 1/3rd of an assistant's paycheck, If you shifted this downward so 500 was the rarest, while like 10-25 credits was the most common, you'd be set. (you could totally fill in the gaps with coins if the paper cash increments are too wide).

Also, completely optional, but I would appreciate it if you added when player's got mail cash to econ logging just so I can keep tabs on it.